Dockerfile.parity

```diff
@@ -19,4 +19,6 @@ WORKDIR /
 FROM keydonix/parity-instantseal
 COPY --from=builder /parity/chains /parity/chains
 
+COPY --from=builder /maker-deployer/output/contracts /contracts
```

Comment:
What is the idea here?  Just so that the user can `docker image cp` the original contract out if they wanted?